### PR TITLE
Add theme categories to web store docs.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: site/en/docs/webstore/best_practices/index.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Include all the [required images][cws-images] (icon, tile, marquee, and screensh
 
 ### Choose your extension's category well {: #choose-category-well}
 
-The developer console lets you specify a category for each extension. Choose the most appropriate category:
+The developer console asks you specify a category for your extension. Choose the most appropriate category:
```
Comment (oliverdunk) [inline site/en/docs/webstore/best_practices/index.md:69]:
```suggestion
The developer console requires you to specify a category for your extension. Choose the most appropriate category:
```

File: site/en/docs/webstore/best_practices/index.md [hunk 2]
```diff
@@ -150,6 +150,47 @@ Social & Communications
...
+: Themes in collections curated by Google.
+
+Animals
+: Themes featuring your favorite animals.
+
+Art & Design
+: Themes build merely to make your browser look pretty.
+
+Cars
+: Themes featuring current and classic cars.
```
Comment (oliverdunk) [inline site/en/docs/webstore/best_practices/index.md:167]: Did these come from some official, already-public definitions or is there scope to change them? Some of them feel a bit weird, e.g with this one - what if I want to feature future cars? Also what if I'm just using the color theme of a car brand but not actually featuring a specific car? They seem too limiting, and I think something like "Themes relating to cars, such as classic or current cars" would be better.

**Review by oliverdunk — CHANGES_REQUESTED** [review]

File: site/en/docs/webstore/best_practices/index.md [hunk 3]
```diff
@@ -150,6 +150,47 @@ Social & Communications
...
 
+### Choose your theme's category well {: #choose-theme-category-well}
+
+The developer console also asks you specify a category for your theme. Choose the most appropriate category:
+
+Artist Series
+: Themes in collections curated by Google.
+
+Animals
+: Themes featuring your favorite animals.
```
Comment (oliverdunk) [inline site/en/docs/webstore/best_practices/index.md:161]:
```suggestion
: Themes inspired by animals.
```

File: site/en/docs/webstore/best_practices/index.md (hunk 2 above)
Comment (oliverdunk) [inline site/en/docs/webstore/best_practices/index.md:167]:
```suggestion
: Themes relating to cars, such as current and classic cars.
```

File: site/en/docs/webstore/best_practices/index.md [hunk 4]
```diff
@@ -150,6 +150,47 @@ Social & Communications
...
+: Themes featuring current and classic cars.
+
+Colors
+: Themes that skin your brwoser in custom colors.
+
+Dark & Black
+: Themes that feature dark colors and imagry.
+
+Entertainment
+: Themes with images of popular television and film franchises.
```
Comment (oliverdunk) [inline site/en/docs/webstore/best_practices/index.md:176]:
```suggestion
: Themes inspired by popular entertainment, such as television and film franchises.
```

File: site/en/docs/webstore/best_practices/index.md [hunk 5]
```diff
@@ -150,6 +150,47 @@ Social & Communications
...
+: Themes that feature popular video games and anime.
+
+Minimalist
+: Themes that simplify the look of your browser.
+
+Nature & Landscapes
+: Themes that feature the great outdoors.
+
+Space
+: Themes that show images of outer space.
```
Comment (oliverdunk) [inline site/en/docs/webstore/best_practices/index.md:188]:
```suggestion
: Themes inspired by space.
```

File: site/en/docs/webstore/best_practices/index.md (hunk 2 above)
Comment (oliverdunk) [reply site/en/docs/webstore/best_practices/index.md:167]: Got it. I left some suggestions :)

**Review by oliverdunk — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/mv3/themes/index.md [hunk 6]
```diff
@@ -3,13 +3,15 @@ layout: "layouts/doc-post.njk"
 title: "What are themes?"
 seoTitle: "Chrome Extensions: What are themes?"
 date: 2012-09-18
-updated: 2022-07-14
+updated: 2023-08-28
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/themes/index.md:6]:
```suggestion
updated: 2023-09-29
```

File: site/en/docs/webstore/best_practices/index.md [hunk 7]
```diff
@@ -3,7 +3,7 @@ layout: "layouts/doc-post.njk"
 title: Best Practices
 seoTitle: Best Practices for Chrome extensions
 date: 2017-08-30
-updated: 2023-08-24
+updated: 2023-08-29
```
Comment (jpmedley) [inline site/en/docs/webstore/best_practices/index.md:6]:
```suggestion
updated: 2023-09-29
```

**Comment by jpmedley** [comment]
@oliverdunk PTAL

**Review by oliverdunk — APPROVED** [review]
Left one nit, otherwise LGTM. Thanks!

**Review by AmySteam — APPROVED** [review]
Thanks for this Joe.

I left one question. But otherwise LGTM.

**Pull request merged**